community/py-lz4/APKBUILD

```diff
@@ -1,40 +1,53 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=py-lz4
-pkgver=0.7.0
+pkgver=0.8.2
 _pkgname=${pkgname#py-}
 pkgrel=0
 pkgdesc="LZ4 Bindings for Python"
 url="https://pypi.python.org/pypi/lz4"
 arch="all"
 license="BSD"
-depends="python"
-depends_dev=""
-makedepends="python-dev py-setuptools lz4-dev"
-install=""
-subpackages=""
-source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz
-	01-system_lz4.patch
-	02-tests.patch
-	"
-
+makedepends="python2-dev python3-dev py-setuptools lz4-dev"
+subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2 $pkgname-tests:_tests"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
 builddir="$srcdir"/$_pkgname-$pkgver
+
 build() {
 	cd "$builddir"
-	python setup.py build || return 1
+	python2 setup.py build || return 1
+	python3 setup.py build || return 1
 }
 
 package() {
+	mkdir -p "$pkgdir"
+}
+
+_py() {
+	local python=$1
+	pkgdesc="$pkgdesc - $python"
+	depends=$2
```

Comment:
What is `$2` doing here? You never pass the second argument to this function. There should be `depends="$python"`.